src/metabase/api/common.clj

```diff
@@ -326,6 +329,20 @@
                                                (wrap-response-if-needed
                                                 (do ~@body))))))))
 
+(defmacro defendpoint-async-schema
```

Comment:
write this to be asymmetry with `defendpoint-schema` and `defendpoint`

Reply:
Nice catch